sdk/cosmosdb/cosmos/samples/MultiRegionWrite/package-lock.json

```diff
@@ -1,272 +0,0 @@
-{
```

Comment:
@HarshaNalluru  Any idea why this file is being deleted in this PR?

Reply:
Not sure why it is present in master. I'll retain it for this PR.

Reply:
`package-lock` file is included in the list of files to be ignored in `.gitignore` file.
While merging master into feature/storage branch, `package-lock.json` got ignored.
I'll retain it in this PR since it is unrelated to this PR and create an issue to remove it from master later on.

Reply:
yeah it has been removed in master branch I believe.

Reply:
Added it back.
No, it is supposed to be removed in the master but not removed yet.

Reply:
nvm it still exists in master

Reply:
#5597